<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filtration Complex</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>921</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Discrete Filter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>331</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Signal generation parameters</string>
      </property>
      <widget class="QLineEdit" name="noiseSigmaLineEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Sigma of noise</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Proportionality</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="proportionalityLineEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of countings</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="countingsLineEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>90</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>200</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Jump alpha</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="jumpAlphaLineEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>120</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>15</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Jump alpha</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="jumpAlphaLineEdit_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>150</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>15</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="simpleGeneratorButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="simpleFilterButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>0</y>
        <width>251</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Filtration methods</string>
      </property>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Empirical Bayes method</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Generalized coordinates method</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Two-criteria method</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="DetailsButton_2">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>190</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Details</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Infinite Filter</string>
     </attribute>
     <widget class="QGroupBox" name="switchingRegimeGroupbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>631</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Signal generation parameters</string>
      </property>
      <widget class="QLineEdit" name="lowerSigmaLineEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Lower Sigma</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>h (size of sigma step)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sigmaStepLineEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of sigma intervals</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="numberOfIntervalsLineEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>90</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Volatility</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="volatilityLineEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>120</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>0.2</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="exitConditionLineEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>150</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>T (exit condition)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="discretizationStepLineEdit">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>30</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>0.01</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>30</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Discretization step</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="startingValueMuLineEdit">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>60</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="startingValueSigmaLineEdit">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>90</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>60</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Starting value mu</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>90</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Starting value sigma</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="noiseParamLineEdit">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>120</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="signalParamLineEdit">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>150</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>120</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Noise generator param</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>150</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Signal generator param</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="switchingRegimeGeneratorButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="switchingRegimeFilterButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="graphicsGroupBox">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>0</y>
        <width>121</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Graphs</string>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Original signal</string>
       </property>
       <widget class="QCheckBox" name="signalSigmaCheckBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>87</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Sigma</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="signalNoiseCheckBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Noise</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="signalCheckBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>87</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Signal</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QCheckBox" name="filteredSignalCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Filtered signal</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="estimationCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Estimation</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>0</y>
        <width>131</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Filtration</string>
      </property>
      <widget class="QSpinBox" name="threadsSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>120</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of threads</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="DetailsButton">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>190</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Details</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>921</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiltraton_Complex">
    <property name="title">
     <string>Filtraton Complex</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDetails"/>
   </widget>
   <addaction name="menuFiltraton_Complex"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDetails">
   <property name="text">
    <string>Filtration Details</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>/Users/ilyamisyura/Qt/jumpFilter/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
